types/titanium/index.d.ts

```diff
@@ -9584,7 +9584,7 @@ declare namespace Titanium {
 			/**
 			 * Replaces an existing section.
 			 */
-			replaceSectionAt(sectionIndex: number, section: Titanium.UI.ListSection, animation: ListViewAnimationProperties): void;
+			replaceSectionAt(sectionIndex: number, section: Titanium.UI.ListSection, animation?: ListViewAnimationProperties): void;
```

Comment:
@ffMathy Please make sure that is reflected in our docs as well. Otherwise this change will be overwritten the next we generate the typings from docs.